ui/src/authors/containers/DetailPage.jsx

```diff
@@ -106,14 +107,19 @@ class DetailPage extends Component {
                 <ExperimentList experiments={experiments} />
               </div>
               <div className="mt3">
-                <Row>
+                <Row gutter={20}>
```

Comment:
did check this for multiple screens, because you may need to set differently for screen sizes?

Reply:
Yes, I did and it seems to look good. On mobile the new row with the table is below the positions so the gutter doesn't affect it